cypress/page/address-step.page.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable require-jsdoc */
```

Comment:
Try to avoid disabling eslint

Reply:
If I don't disable require-jsdoc it shows me many warnings like this one in every Page Object File 
Missing JSDoc comment

Reply:
![image](https://user-images.githubusercontent.com/34688953/180624724-dd183604-25f8-40cd-b1b6-aa0b34f8fc41.png)
 It is because it is configured to give warnings on the eslint configuration, but as long as it does not gives errors it is not an issue :)